src/test/java/tests/NegativeSmokeTest.java

```diff
@@ -0,0 +1,74 @@
+package tests;
+
+import indices.Food;
+import indices.Pages;
+import indices.Pizza_Options;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.DominoPage;
+import indices.Icons;
+import pages.MainPage;
+import properties.Type;
+
+public class NegativeSmokeTest extends BaseTest {
+
+    @Test (description = "Test to check system behavior using the correct password and incorrect login")
+    public void setIncorrectLoginTest() {
+        MainPage mainPage = new MainPage()
+                .openPage()
+                .isPageOpened()
+                .openEntranceForm()
+                .insertLogin(faker.internet().emailAddress())
+                .insertPassword(props.getKeyProperty(Type.PASSWORD))
+                .clickEntranceButton();
+        Assert.assertEquals(mainPage.getLoginWarningMessage(), "неверный логин или пароль", "The login is correct!");
+    }
+
+    @Test(description = "Test to check system behavior using the correct login and incorrect password")
+    public void setIncorrectPasswordTest() {
+        MainPage mainPage = new MainPage()
+                .openPage()
+                .isPageOpened()
+                .openEntranceForm()
+                .insertLogin(props.getKeyProperty(Type.LOGIN))
```

Comment:
create separate file or smth to separate test data from other properties

Reply:
done